apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -309,6 +311,37 @@ const WorkshopForm = React.createClass({
     }
   },
 
+  shouldRenderRegionalPartnerSelect() {
+    return window.dashboard.workshop.available_regional_partners.length;
+  },
+
+  renderRegionalPartnerSelect(validation) {
+    if (this.shouldRenderRegionalPartnerSelect()) {
+      const options = window.dashboard.workshop.available_regional_partners.map((partner) => {
+        return (<option key={partner.id} value={partner.id}>{partner.name}</option>);
+      });
+      // we always present an empty placeholder for an optional field
+      const placeHolder = <option />;
+      return (
+        <FormGroup validationState={validation.style.regional_partner_id}>
+          <ControlLabel>Regional Partner</ControlLabel>
+          <FormControl
```

Comment:
oh, given that not all users will be able to see this, having it on its own row makes the experience _way_ nicer for those users for which it's hidden. I'll do that.